src/app/common/grade/grade.component.html

```diff
@@ -1,8 +1,10 @@
-<span class="grade-name">{{ legacy ? grade.legacyName : grade.name }}<ng-container *ngIf="showModifier"> </ng-container>
```

Comment:
ngif on an empty container, is this a mistake?

<ng-container *ngIf="showModifier"> </ng-container>

Reply:
It was there before and I was guessing that it is there because of the space (" ") to seaprate the grade name from the modifier.

But I didn't check that.